taxonomy_fixtures.module

```diff
@@ -34,17 +34,23 @@ function taxonomy_fixtures_import_terms($vocabularies = array()) {
     $data_file = implode('/', $data_file);
     $data_file = file_get_contents($data_file);
     $terms = drupal_json_decode($data_file);
+    
     // Load Vocabulary.
     $vocabulary = taxonomy_vocabulary_machine_name_load($machine_name);
-    foreach ($terms as $key => $term) {
-      $import = new TermFixtureImport(
-        $term,
-        $vocabulary,
-        $config['map'],
-        $config['export_path']
-      );
-      $imports[] = $import;
+    if (!empty($vocabulary)) {
+      foreach ($terms as $key => $term) {
+        $import = new TermFixtureImport(
+          $term,
+          $vocabulary,
+          $config['map'],
+          $config['export_path']
+        );
+        $imports[] = $import;
+      }
     }
+    else {
+	    drupal_set_message(t("Vocabulary - $machine_name not exist"), 'error'); 
```

Comment:
Hi @rahulseth - two things:
1. We can't pass variables inside strings to the t() function, please use placeholders. See [t() documentation](https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/t/7.x)
2. I think the right wording would be "Vocabulary _vocab\_machine\_name_ does not exist."

Thanks!